api/routes/orderRouter.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const { orderController } = require("../controllers");
+const {loginRequired} = require("../utils/auth");
+
+const router = express.Router();
+
+router.get("/", loginRequired, orderController.order);
+router.post("/", loginRequired, orderController.payment);
```

Comment:
```suggestion
router.get("", loginRequired, orderController.order);
router.post("", loginRequired, orderController.payment);
```
REST API에서는 Uniform Interface 원칙에 따라, 엔드포인트 끝에 (`/`)를 넣어서 작성하는 것을 지양하고 있습니다.